### ISSUE TITLE
Include custom editor for .inp files

### ISSUE THREAD

**Issue by IanMayo** [debrief/debrief#1723]
@dinkoivanov has developed a demonstrator for editing .inp files in Debrief:
https://github.com/dinkoivanov/projects/tree/master/editor

But, it needs to be incorporated into the org.mwc.debrief.dis plugin, because I forgot to do it :-)

@theanuradha - I'd like you to incorporate it, please.  But, before you start coding, could you give me a quick list of the steps you'll be taking to incorporate it, just to be sure the task is clear, and that we don't forget anything.

I can think of these steps:
* copy sources, refactor to correct name space
* inject dinko's plugin.xml code into the org.mwc.debrief.dis plugin.xml
* ensure all dependencies are added to manifest.mf
* register .inp as a new file-type, so *.inp*  files in the navigator are shown with the DIS icon: https://github.com/debrief/debrief/blob/master/org.mwc.debrief.dis/icons/16px/dis_icon.png

**Comment by IanMayo** [comment]
Here's an example of content for a *.inp* file:

````
// params as follows:
// IP address, Port Id, network mode, 
// time interval, num participants, num scenario cycles
239.1.2.3 62040 multicast 1000 12 15

<< some other separator >>
````

**Comment by theanuradha** [comment]
<img width="884" alt="inp-editor" src="https://cloud.githubusercontent.com/assets/535713/14818369/5edd8a02-0bda-11e6-9b9e-3df1902f74e0.png">

**Comment by theanuradha** [comment]
@IanMayo I have added/ported and make sure source header comment in place too.

**Comment by IanMayo** [comment]
Fixed in #1731
